content/zh-cn/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -15,109 +15,115 @@ approvers:
 
 <!--
 This tutorial provides an introduction to managing applications with
-[StatefulSets](/docs/concepts/workloads/controllers/statefulset/). It
-demonstrates how to create, delete, scale, and update the Pods of StatefulSets.
+{{< glossary_tooltip text="StatefulSets" term_id="statefulset" >}}.
+It demonstrates how to create, delete, scale, and update the Pods of StatefulSets.
 -->
-
-本教程介绍了如何使用 [StatefulSets](/zh/docs/concepts/workloads/controllers/statefulset/) 来管理应用。
+本教程介绍了如何使用
+{{< glossary_tooltip text="StatefulSets" term_id="statefulset" >}}
+来管理应用。
 演示了如何创建、删除、扩容/缩容和更新 StatefulSets 的 Pods。
 
-
-
 ## {{% heading "prerequisites" %}}
 
-
 <!--
 Before you begin this tutorial, you should familiarize yourself with the
-following Kubernetes concepts.
+following Kubernetes concepts:
 -->
-
 在开始本教程之前，你应该熟悉以下 Kubernetes 的概念：
 
-* [Pods](/zh/docs/concepts/workloads/pods/)
-* [Cluster DNS](/zh/docs/concepts/services-networking/dns-pod-service/)
-* [Headless Services](/zh/docs/concepts/services-networking/service/#headless-services)
-* [PersistentVolumes](/zh/docs/concepts/storage/persistent-volumes/)
+<!--
+* [Pods](/docs/concepts/workloads/pods/)
+* [Cluster DNS](/docs/concepts/services-networking/dns-pod-service/)
+* [Headless Services](/docs/concepts/services-networking/service/#headless-services)
+* [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
+* [PersistentVolume Provisioning](https://github.com/kubernetes/examples/tree/master/staging/persistent-volume-provisioning/)
+* [StatefulSets](/docs/concepts/workloads/controllers/statefulset/)
+* The [kubectl](/docs/reference/kubectl/kubectl/) command line tool
+-->
+* [Pods](/zh-cn/docs/concepts/workloads/pods/)
```

Comment:
Should it be `Pods` or `Pod`?

The [target link](https://kubernetes.io/zh-cn/docs/concepts/workloads/pods/) is titled as `Pods`, with all content written in `Pod`.

Reply:
As far as I understand, `Pod` is often correct if localized to Chinese. We never use `-s` to describe plural instead of using quantifiers such as 个, 台, 张, 批 before a noun ...